src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -621,7 +621,7 @@ void DlgPrefSound::updateAudioBufferSizes(int sampleRateIndex) {
  * just changes and we need to display new devices.
  */
 void DlgPrefSound::refreshDevices() {
-    if (m_config.getAPI() == "None") {
+    if (m_config.getAPI() == MIXXX_NOSOUNDAPI_STRING) {
```

Comment:
This breaks existing configurations. 
If this is the user data "None" should be defined as constant as well. Something like kNoApiProgramaticString or such.

Reply:
I'll use existing `SoundManagerConfig::kDefaultAPI` so that string is defined in one place only
https://github.com/mixxxdj/mixxx/blob/5fe525fe73450039ef4cef2067830705244aed26/src/soundio/soundmanagerconfig.cpp#L14